.docker/default.conf

```diff
@@ -0,0 +1,10 @@
+server {
+    listen       8080;
+    server_name  localhost;
```

Comment:
Not a good idea, we want this to answer any `server_name`. I know this will work as nginx will use the first server block it finds but I prefer us to be explicit so either use lumx public name or remove the server_name and use the `default_server` helper.

```
    listen       8080  default_server;
```